libsql-server/src/hrana/ws/conn.rs

```diff
@@ -206,16 +206,18 @@ async fn handle_client_msg(conn: &mut Conn, client_msg: proto::ClientMsg) -> Res
 }
 
 async fn handle_hello_msg(conn: &mut Conn, jwt: Option<String>) -> Result<bool> {
-    let hello_res = match conn.session.as_mut() {
-        None => {
-            session::handle_initial_hello(&conn.server, conn.version, jwt, conn.namespace.clone())
-                .await
-                .map(|session| conn.session = Some(session))
-        }
-        Some(session) => {
-            session::handle_repeated_hello(&conn.server, session, jwt, conn.namespace.clone()).await
-        }
-    };
+    let auth = session::handle_hello(&conn.server, jwt, conn.namespace.clone()).await;
```

Comment:
This is already in main as well, no? https://github.com/tursodatabase/libsql/blob/main/libsql-server/src/hrana/ws/conn.rs#L209

It generally makes maintainer's work much easier if PRs are rebased and contain only new changes. I would be grateful if you could try to make sure that your PRs are rebased and clean. Thank you!